### PR TITLE
Overview3

### PR DESCRIPTION
**Pull request by EJLEE808**
Implemented Click Captures 
Implemented Zoom In Feature General Bug Fixin' around Modal

### CODE REVIEW

File: client/src/components/overview/components/features.jsx [hunk 1]
```diff
@@ -1,41 +1,58 @@
...
 
-  render() {
-    return (
-      <>
-        <div>Features Here</div>
-        {this.makeFeatures()}
-      </>
-    )
+
+let Features = (props) => {
```
Comment (Taniel-P) [inline client/src/components/overview/components/features.jsx:37]: I think this is the beginning of the functional component Features? You can use const here if that's the case.

File: client/src/components/overview/components/features.jsx [hunk 2]
```diff
@@ -1,41 +1,58 @@
...
+  if (props.features === undefined) {
+    return <div></div>
   }
-}
+  let featuresArr = [];
+  let i = 0;
```
Comment (Taniel-P) [inline client/src/components/overview/components/features.jsx:42]: Will i be 0 every time this function is ran?

Comment (EJLEE808) [reply client/src/components/overview/components/features.jsx:42]: Yes

File: client/src/components/overview/overview.jsx [hunk 3]
```diff
@@ -87,7 +98,119 @@ class Overview extends React.Component {
...
+  }
+
+
+  consoleLogAfterSETSTATE(test) {
+    this.setState({ test }, () => {
+      console.log('The Second Argument of setState should be a function that runs')
+    })
+  }
+
+  clickModal() {
```
Comment (Taniel-P) [inline client/src/components/overview/overview.jsx:113]: Nice on all these!

**Review by Taniel-P — APPROVED** [review]
Looking good!

**Pull request merged**